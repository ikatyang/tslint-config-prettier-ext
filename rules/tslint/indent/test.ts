if (true) {
	console.log("1-tab");
}
