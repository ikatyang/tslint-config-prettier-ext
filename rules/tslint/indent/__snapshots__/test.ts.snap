// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (indent) /rules/tslint/indent/test.ts[2, 1]: 2 space indentation expected

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
if (true) {
	console.log(\\"1-tab\\");
}

======
if (true) {
  console.log(\\"1-tab\\");
}

>>>>>> after
"
`;
